e2e/app/workspace-card.ts

```diff
@@ -31,10 +31,11 @@ export default class WorkspaceCard extends BaseElement {
     return resourceCards;
   }
 
-  static async getAnyCard(page: Page): Promise<WorkspaceCard> {
+  static async getAnyCard(page: Page): Promise<WorkspaceCard | null> {
```

Comment:
I don't think this change makes sense. An async function should only ever resolve the Promise if the expected return value is there; in other cases, an error should be returned so the Promise is rejected.

I also can't quite tell why this change is being made – all other changes in this PR are changing how the response to getAllCards is handled, not getAnyCard. Is there a particular test case which required this change?